run-tox-env.yml

```diff
@@ -2,6 +2,7 @@ parameters:
   libraries: []
   envs: []
   coverage: false
+  pytest: true
```

Comment:
This is "true" but feels like it should really be "false", changing it would probably break some stuff though.